crates/sui-rpc-benchmark/src/json_rpc/runner.rs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// This module implements the JSON RPC benchmark runner.
+/// The main function is `run_queries`, which runs the queries concurrently
+/// and records the metrics.
+use anyhow::Result;
+use std::{
+    collections::HashMap,
+    sync::{Arc, Mutex},
+    time::Instant,
+};
+use sui_indexer_alt_framework::task::TrySpawnStreamExt;
+use tokio::time::timeout;
+
+use super::request_loader::JsonRpcRequestLine;
+use crate::config::BenchmarkConfig;
+
+#[derive(Clone, Default)]
+pub struct MethodMetrics {
```

Comment:
Stats is probably a better name since this is not the same as other metrics that are sent to grafana.